### PR TITLE
Add UG and clean up outputs for Tag and Untag

### LINKED ISSUE #97: Update User Guide for Tag/Untag Command

### PR DESCRIPTION
**Pull request by kennycjy**
Relatively straightforward.

TODO:
Show in the UG that it is possible to do multiple arguments at once for both indexed and index-less uses of the commands.

Closes #97

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -155,38 +155,85 @@ This command removes the module stated in the command (“CS2103T”) from the l
...
 ### Tagging a module to contact : `tag`
 
-Adds a module tag to an existing contact.
-Format: `tag n/NAME m/MODULE_TAG`
-Example of usage: `tag n/John Doe m/CS2103T`
+Adds module tag(s) to an existing contact.
+
+Formats: 
+* `tag CONTACT_INDEX m/MODULE_TAG`
+* `tag m/MODULE_TAG`
```
Comment (kennycjy) [inline docs/UserGuide.md:162]: Basically added two formats for the command.
I expect to add two more in the future to showcase multiple commands.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -155,38 +155,85 @@ This command removes the module stated in the command (“CS2103T”) from the l
...
 
-Example of usage: `untag n/John Doe m/CS2103T`
-Context:
+Formats: 
+* `untag CONTACT_INDEX m/MODULE_TAG`
+* `untag m/MODULE_TAG`
+
+Example of usage: `untag 3 m/CS2103T`
+```
+Name: John Doe (User)
```
Comment (russelltankaimin) [inline docs/UserGuide.md:207]: As per CS2101 recommendations, lets not use John Doe and change the name up abit.

File: docs/UserGuide.md (hunk 1 above)
Comment (russelltankaimin) [reply docs/UserGuide.md:162]: Ok.

**Review by russelltankaimin — APPROVED** [review]
John Doe should be changed but in a future PR. We should standardise it in a later PR in terms of our Characters. Thanks. LGTM for now.

**Pull request merged**